src/app/app.module.ts

```diff
@@ -11,24 +11,27 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
  */
-import { BrowserModule } from '@angular/platform-browser';
-import { NgModule } from '@angular/core';
-import { FormsModule } from '@angular/forms';
-import { AppComponent } from './app.component';
-import { AboutComponent } from './about.component';
-import { SituationModule } from './situations/situation.module';
-import { AppRoutingModule } from './app-routing.module';
-import { PageNotFoundComponent } from './page-not-found.component';
-import { ApplicationManagementModule } from './application-management/application-management.module';
-import { RepositoryModule } from './repository/repository.module';
-import { ConfigurationModule } from './configuration/configuration.module';
-import { CoreModule } from './core/core.module';
-import { AccordionModule, CardModule, ConfirmDialogModule, GrowlModule, PanelModule } from 'primeng/primeng';
-import { NgReduxModule } from '@angular-redux/store';
-import { NgReduxRouterModule } from '@angular-redux/router';
-import { StoreModule } from './store/store.module';
-import { HttpClientModule } from '@angular/common/http';
-import { TabMenuModule } from 'primeng/tabmenu';
+import {BrowserModule} from '@angular/platform-browser';
```

Comment:
Please revert the whitespace changes (import the corresponding scheme to IntelliJ)